kubeflow/trainer/types/types.py

```diff
@@ -110,6 +110,45 @@ class TorchTuneInstructDataset:
     column_map: Optional[dict[str, str]] = None
 
 
+@dataclass
+class LoraConfig:
+    """Configuration for the LoRA/QLoRA/DoRA.
+    REF: https://docs.pytorch.org/torchtune/main/tutorials/memory_optimizations.html#parameter-efficient-fine-tuning-peft
```

Comment:
I think, this link get updated: https://meta-pytorch.org/torchtune/main/tutorials/memory_optimizations.html